Quadriga/src/main/java/edu/asu/spring/quadriga/dto/AboutTextDTO.java

```diff
@@ -0,0 +1,70 @@
+package edu.asu.spring.quadriga.dto;
+
+import java.io.Serializable;
+
+import javax.persistence.Basic;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+/**
+ * DTO for about project text of project's public website.
+ * 
+ * @author Rajat Aggarwal
+ */
+
+@Entity
+@Table(name = "tbl_about_text")
+public class AboutTextDTO implements Serializable {
+
+    private static final long serialVersionUID = 1103019135158917211L;
+
+    @Id
+    private String Id;
+
+    public String getId() {
+        return Id;
+    }
+
+    public void setId(String id) {
+        Id = id;
+    }
```

Comment:
all your getters/setters should be together and all your variable declarations should be together. these methods should be moved down.